gossip/client/hubclient.go

```diff
@@ -165,12 +165,14 @@ func (c *HubClient) GetProofByHash(ctx context.Context, hash []byte, treeSize ui
 	return &rsp, nil
 }
 
+// getEntriesRsp retrieves the given range of entries, returning an HTTP status code,
+// HTTP response body and slice of TLS-encoded TimetStampedEntries, or an error.
 func (c *HubClient) getEntriesRsp(ctx context.Context, start, end int64) (int, []byte, [][]byte, error) {
 	if start < 0 {
-		return -1, nil, nil, errors.New("start should be >= 0")
+		return 0, nil, nil, errors.New("start should be >= 0")
```

Comment:
Should these http response codes be using `http` constants? 

```suggestion
		return http.StatusBadRequest, nil, nil, errors.New("start should be >= 0")
```

Reply:
Ah, [missing context](https://github.com/google/trillian-examples/pull/26#discussion_r224489561)